elements/rh-secondary-nav/demo/demo.css

```diff
@@ -18,7 +18,7 @@ rh-secondary-nav-menu#custom-grid::part(sections) {
 }
 
 /* Dark demo with long logo text */
-@media screen and (min-width: 768px) {
+@media screen and (min-width: 358px) {
```

Comment:
Is this a permanent change or just one made for the demo purposes of this bug?

Reply:
This is just an implementation example, the change has nothing to do with this PR specifically.  Just correcting a missed opportunity in the demo to show how a long text logo can have a custom breakpoint to break into 3 lines.